page-cabinet-template.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+Template Name: Cabinet documents
```

Comment:
@sdwilkes, after our brainstorm:neckbeard:, i have changed the name to 'Homepage'. It is generic, so we can use it for other projects in the future.